fdbserver/workloads/ApiCorrectness.actor.cpp

```diff
@@ -98,6 +110,12 @@ struct ApiCorrectnessWorkload : ApiWorkload {
 	// Maximum time to reset DB to the original state
 	double resetDBTimeout;
 
+	// Validate data at-rest encryption guarantees
+	int validateEncryptionAtRest;
+
+	int numTenants;
```

Comment:
Nit: remove (i think this is unused) same with `Optional<Reference<Tenant>> tenant` below

Reply:
Yeah, seems like the fallout from my previous attempts. If CI is OK, will do it as a followup PR. 
Thanks